lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -139,18 +139,15 @@ def request_json_matches(required_attributes)
         end
       end
 
-      def publishing_api_has_fields_for_document(format, items, fields)
+      def publishing_api_has_content(items, params = {})
```

Comment:
Can you provide documentation on how to use this method please?
Follow the convention [used here](https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/test_helpers/publishing_api_v2.rb#L183) so we can read the documentation here: http://www.rubydoc.info/gems/gds-api-adapters

Thank you